docs/op-guide/deploy-tikv-using-ansible.md

```diff
@@ -0,0 +1,567 @@
+---
+title: Install and Deploy TiKV Using Ansible
+summary: Use TiDB-Ansible to deploy a TiKV cluster on multiple nodes.
+category: operations
+---
+
+# Install and Deploy TiKV Using Ansible
+
+This guide describes how to install and deploy TiKV using Ansible. Ansible is an IT automation tool that can configure systems, deploy software, and orchestrate more advanced IT tasks such as continuous deployments or zero downtime rolling updates.
+
+[TiDB-Ansible](https://github.com/pingcap/tidb-ansible) is a TiDB cluster deployment tool developed by PingCAP, based on Ansible playbook. TiDB-Ansible enables you to quickly deploy a new TiKV cluster which includes PD, TiKV, and the cluster monitoring modules.
+
+> **Note:** For the production environment, use TiDB-Ansible to deploy your TiKV cluster. If you only want to try TiKV out and explore the features, see [Install and Deploy TiKV using Docker Compose](deploy-tikv-using-docker-compose.md) on a single machine.
+
+## Prepare
+
+Before you start, make sure you have:
+
+1. Several target machines that meet the following requirements:
+
+    - 4 or more machines
+
+        A standard TiKV cluster contains 6 machines. You can use 4 machines for testing.
+
+    - CentOS 7.3 (64 bit) or later with Python 2.7 installed, x86_64 architecture (AMD64)
+    - Network between machines
+    
+    > **Note:** When you deploy TiKV using Ansible, use SSD disks for the data directory of TiKV and PD nodes. Otherwise, it cannot pass the check. For more details, see [Software and Hardware Requirements](https://github.com/pingcap/docs/blob/master/op-guide/recommendation.md).
```

Comment:
I think we can have a better reason than this. Maybe "Otherwise, the system will not perform well."